draft-packages/guidance-block/docs/GuidanceBlock.stories.tsx

```diff
@@ -114,6 +121,24 @@ const PROPS: GuidanceBlockProps = {
   },
 }
 
+const CustomContent = () => (
+  <>
+    <Box mb={0.75}>
+      <Tag variant="statusLive" size="small">
+        Early Access
+      </Tag>
+    </Box>
+    <div className={styles.headingWrapper}>
```

Comment:
This should be replaced with `Box` as well, because we don't want to encourage people reaching into the CSS file.
The class is using  the `spacing-md` token, so it would probably be `<Box mb={1}>...`